custom_components/tuya_local/devices/solar_inverter_SPD800.yaml

```diff
@@ -0,0 +1,117 @@
+name: Micro Inverter Stepuptech SPD-800
```

Comment:
This device already exists in the config file list, there is no need for this new file